core/src/test/scala/org/apache/spark/ui/UISuite.scala

```diff
@@ -398,8 +386,8 @@ class UISuite extends SparkFunSuite {
     }
   }
 
-  test("SPARK-45522: Jetty 10 and above shouuld return status code 302 with correct redirect url" +
-    " when request URL ends with a context path without trailing '/'") {
+  test("SPARK-34449: Jetty 9.4.35.v20201120 and later no longer return status code 302 " +
+    " and handle internally when request URL ends with a context path without trailing '/'") {
```

Comment:
I'm not sure why do we change this in this PR. Could you spin-off this in order to merge before this PR?